webdriver-spec.html

```diff
@@ -3814,7 +3814,7 @@
 
 <p>A <dfn data-lt="pointer-interactable">pointer-interactable element</dfn>
  is defined to be the first non-transparent <a>element</a>,
- defined by the paint order found at the centre point
+ defined by the paint order found at the centerpoint
```

Comment:
Well “centerpoint” is actually fairly established, but I’m very happy to continue using two words.